### ISSUE TITLE
No error message for JSDoc type parsing

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/typescript-go#1880]
In a JS file:

```js
// @ts-check

/** @type {@import("something").Type} */
let x;
```

Currently, no error is reported for this file.

Expected:

```
wat.mjs:3:12 - error TS1110: Type expected.

3 /** @type {@import("something").Type} */
             ~
```
